microservices-orders/docs/index.rst

```diff
@@ -0,0 +1,578 @@
++-------------------------------------+-----------------------------------+-----------------------+
+| OrdersService                       | -                                 | `orders`              |
++-------------------------------------+-----------------------------------+-----------------------+
+| PostOrdersAndPayments               | -                                 | `payments`            |
++-------------------------------------+-----------------------------------+-----------------------+
+| AddInventory                        | -                                 | `warehouse-inventory` |
++-------------------------------------+-----------------------------------+-----------------------+
+
+
+End-to-end Streaming ETL
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+This demo showcases an entire end-to-end streaming ETL deployment, built around the microservices described above.
+It is build on the |cp|, including:
+
+* JDBC source connector: reads from a sqlite database that has a table of customers information and writes the data to a Kafka topic, using Connect transforms to add a key to each message
+* Elasticsearch sink connector: pushes data from a Kafka topic to Elasticsearch
+* KSQL: creates streams and tables and joins data from a STREAM of orders with a TABLE of customer data
+
++-------------------------------------+-----------------------+-------------------------+
+| Other Clients                       | Consumes From         | Produces To             |
++=====================================+=======================+=========================+
+| JDBC source connector               | DB                    | `customers`             |
++-------------------------------------+-----------------------+-------------------------+
+| Elasticsearch sink connector        | `orders`              | ES                      |
++-------------------------------------+-----------------------+-------------------------+
+| KSQL                                | `orders`, `customers` | KSQL streams and tables |
++-------------------------------------+-----------------------+-------------------------+
+
+
+==============
+Pre-requisites
+==============
+
+
+Reading
+~~~~~~~
+
+You will get a lot more out of this tutorial if you have first learned the concepts which are foundational for this tutorial.
+To learn how service-based architectures and stream processing tools such as Apache Kafka® can help you build business-critical systems, we recommend:
+
+* If you have lots of time: `Designing Event-Driven Systems <https://www.confluent.io/designing-event-driven-systems>`__, a book by Ben Stopford.
+* If you do not have lots of time: `Building a Microservices Ecosystem with Kafka Streams and KSQL <https://www.confluent.io/blog/building-a-microservices-ecosystem-with-kafka-streams-and-ksql/>`__ or `Build Services on a Backbone of Events <https://www.confluent.io/blog/build-services-backbone-events/>`__.
+
```

Comment:
nit: maybe also worth adding the entry index of Streams docs?

Reply:
Ack